.github/workflows/mypy_primer.yml

```diff
@@ -32,6 +32,11 @@ jobs:
       - uses: actions/setup-python@v4
         with:
           python-version: "3.10"
+          cache: 'pip'
+          cache-dependency-path: |
+            typeshed_to_test/requirements-tests.txt
+            typeshed_to_test/stubs/**/@tests/requirements-stubtest.txt
+            typeshed_to_test/stubs/**/METADATA.toml
```

Comment:
I'm not sure this will actually speed anything up. The only thing this workflow directly pip-installs is `mypy_primer`, which has 0 dependencies. `mypy_primer` will then dynamically install loads of things during the course of the script's execution, but I don't think it'll ever install anything from typeshed's `requirements-tests.txt` file, any `METADATA.toml` files, or any `requirements-stubtest.txt` files.

Maybe just get rid of the changes to this workflow file, for now?

Reply:
It would be more sensible to base the cache key on `mypy_primer.py`. But I can't really specify that. mypy_primer downloads so much it would make sense to cache those. But then how do we specify that the cache should be invalidated? Here I tried to target everything that may suggest a need to invalidate (ie version bump or dependency change of the stub).

Anyway, this can be reconsidered, I'll revert for now.